src/rajAJun20/FibonacciNumbers.java

```diff
@@ -0,0 +1,22 @@
+package rajAJun20;
+
+public class FibonacciNumbers {
+	
+	void findFibonacciNumbers(int input) {
+			int sum = 0;
+			int firstNum = 0  , nextNum = 1 ;
+			System.out.println("Fibonacci series for input " +input+ " is :");
+			for(int index = 0 ; index < input ; index++ ) {
+				sum = firstNum + nextNum;
+				System.out.println(nextNum);
```

Comment:
Here you have to write System.out.print(nextNum); for printing in same line. Println is used  for printing in nextline.